<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A CS123 Final</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="View" name="view" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>35</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>451</width>
       <height>541</height>
      </rect>
     </property>
     <widget class="QSlider" name="ka_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="ka_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ka</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ka_lineEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="kd_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>kd</string>
      </property>
     </widget>
     <widget class="QSlider" name="kd_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="kd_lineEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ks_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ks</string>
      </property>
     </widget>
     <widget class="QSlider" name="ks_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="ks_lineEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="kr_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>kr</string>
      </property>
     </widget>
     <widget class="QSlider" name="kr_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="kr_lineEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="ambient_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ambient Color</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ambient_lineEdit_g">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ambient_lineEdit_b">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ambient_lineEdit_r">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Realtime 3D Fractal Rendering</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QWidget</extends>
   <header>view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
